src/core/factcheck_plus_worker.py

```
@@ -492,6 +518,20 @@ def _on_claim_progress(self, index: int, total: int, claim_id: str) -> None:
         self.stage_changed.emit(f"Recherchiere Claim {index}/{total} …", 5, TOTAL_STAGES)
         self.claim_progress.emit(index, total)
 
+    def _finalize_cancelled(self) -> None:
+        """Gemeinsamer Abschluss bei Abbruch VOR/IN einer frühen Stufe (S1–S4).
```

Ruff (0.16.3)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)